pkg/kube/apisix/apis/config/v1/zz_generated.register.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // GroupName specifies the group name used to register the objects.
-const GroupName = "apisix.apache.org"
+const GroupName = "config"
```

Comment:
I guess this is wrong. Leave the group option empty to let it use the domain (apisix.apache.org) directly.